tests/org/mozilla/io/TestLocalMsgProtocol.java

```diff
@@ -380,24 +323,11 @@ public void test(TestHarness th) {
 
             // Scenario 2
             serverCreateThread.start();
-            synchronized (openLock) {
-                while (!serverCreated) {
-                    openLock.wait();
-                }
-            }
+            serverCreateThread.join();
             clientThread.start();
-            synchronized (openLock) {
-                while (!clientCreated) {
-                    openLock.wait();
-                }
-            }
```

Comment:
Scenario 2 was:
1) Server creates a localmsg server
2) Client opens a localmsg server
3) Server calls acceptAndOpen

With this change, 3 could happen before 2.